test/e2e/tests/settings/address-book.spec.js

```diff
@@ -111,25 +110,15 @@ describe('Address Book', function () {
 
         await driver.clickElement('[data-testid="page-container-footer-next"]');
 
-        const recipientUsername = await driver.findElement({
+        await driver.findElement({
           text: 'Test Name Edit',
           css: '.address-list-item__label',
         });
 
-        assert.equal(
-          await recipientUsername.getText(),
-          'Test Name Edit',
-          'Username is not edited correctly',
-        );
-
-        const recipientAddress = await driver.findElement(
-          '[data-testid="address-list-item-address"]',
-        );
-        assert.equal(
-          await recipientAddress.getText(),
-          shortenAddress('0x74cE91B75935D6Bedc27eE002DeFa566c5946f74'),
-          'Recipient address is not edited correctly',
-        );
+        await driver.findElement({
```

Comment:
is there a reason we use `findElement` but not `waitForSelector`?

Reply:
I was thinking about this too.  They are pretty similar, unless you need `visible` or `detached`.  And I actually I believe that `findVisibleElement` is exactly the same as `waitForSelector` with `visible`.

Reply:
hey :wave: thank you for the catch! It was not intended, I missed to change it, as in the original test it was using findElement.

I agree it won't make much difference, and looking for the selector with the text is enough for us, so both would work, but I've updated it to `waitForSelector` just because that's what we've been doing in those cases (it was also recommended by David).

Thank you both!!